Tasks/ArchiveFilesV2/utils.ts

```diff
@@ -0,0 +1,18 @@
+import tl = require("vsts-task-lib/task");
+
+export function reportArchivePlan(files: string[], max: number=10) {
+    var plan: string[] = [];
+    plan.push(tl.loc('FoundNFiles', files.length));
+    if (files.length > 0) {
+        for (var i = 0; i < files.length; i++) {
```

Comment:
nit: could simplify this logic a bit with `for (var i = 0; i < Math.max(files.length, max); i++) {`